### PR TITLE
Update the eBPF profiling schedule entity

### PR DESCRIPTION
**Pull request by mrproliu**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

Because this enhancement is duplicated with `Implement the eBPF profiling query and data collect protocol` in the `CHANGES.MD`, I think we don't need to update the changes file.

image

### CODE REVIEW

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java [hunk 1]
```diff
@@ -83,7 +97,17 @@ private EBPFProfilingAnalyzer getProfilingAnalyzer() {
     }
 
     public List<EBPFProfilingSchedule> queryEBPFProfilingSchedules(String taskId, Duration duration) throws IOException {
-        return getScheduleDAO().querySchedules(taskId, duration.getStartTimeBucket(), duration.getEndTimeBucket());
+        final List<EBPFProfilingSchedule> schedules = getScheduleDAO().querySchedules(taskId, duration.getStartTimeBucket(), duration.getEndTimeBucket());
+        if (CollectionUtils.isNotEmpty(schedules)) {
+            schedules.forEach(p -> {
+                try {
+                    p.setProcess(getMetadataQueryDAO().getProcess(p.getProcessId()));
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java:104]: Please implement multiple get. I am not sure whether process ID is the row ID of the process entity?

Comment (mrproliu) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java:104]: Yes, the process id is the row ID in storage. I will update to use multiple get.

Comment (mrproliu) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java]: Updated.

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/MetadataQueryEsDAO.java [hunk 2]
```diff
@@ -230,6 +230,17 @@ public Process getProcess(String processId) throws IOException {
         return processes.isEmpty() ? null : processes.get(0);
     }
 
+    @Override
+    public List<Process> getProcesses(List<String> processIdList) throws IOException {
```
[the remark this answers is not in the document]
Comment (mrproliu) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/MetadataQueryEsDAO.java:234]: Use that means I need to re-create `IMetricsDAO` by `StorageDAO#newMetricsDao`, and re-mapping `ProcessTraffic` to `Process`(graphql). Currently, we already have storage in graphQL logic, so I just keep the `Process` created in one place. Should I change it?

[the remark this answers is not in the document]
Comment (wu-sheng) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/MetadataQueryEsDAO.java:234]: Yes, I know we have to re-create those objects, but new interfaces mean we are adding more and more burdens to the storage level, which have N implementations. Let's not do that.

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java [hunk 3]
```diff
@@ -83,10 +128,47 @@ private EBPFProfilingAnalyzer getProfilingAnalyzer() {
...
+        process.setServiceName(IDManager.ServiceID.analysisId(serviceId).getName());
+        final String instanceId = traffic.getInstanceId();
+        process.setInstanceId(instanceId);
+        process.setInstanceName(IDManager.ServiceInstanceID.analysisId(instanceId).getName());
+        process.setLayer(Layer.valueOf(traffic.getLayer()).name());
+        process.setAgentId(traffic.getAgentId());
+        process.setDetectType(ProcessDetectType.valueOf(traffic.getDetectType()).name());
+        if (traffic.getProperties() != null) {
+            for (String key : traffic.getProperties().keySet()) {
+                process.getAttributes().add(new Attribute(key, traffic.getProperties().get(key).getAsString()));
```
[the remark this answers is not in the document]
Comment (mrproliu) [reply oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/profiling/ebpf/EBPFProfilingQueryService.java:169]: ignore

File: oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/MetadataQueryEsDAO.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (mrproliu) [reply oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/query/MetadataQueryEsDAO.java]: Updaed.

File: oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java [hunk 4]
```diff
@@ -29,6 +29,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+
```
Comment (wu-sheng) [inline oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/MetadataQuery.java:32]: How is this changed? New code style?

**Review by wu-sheng — APPROVED** [review]
LGTM, a nit only.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**